Cabal/Distribution/Compat/DList.hs

```diff
@@ -0,0 +1,36 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Distribution.Compat.DList
+-- Copyright   :  (c) Ben Gamari 2015-2019
```

Comment:
copied from `Cabal/Distribution/Lex.hs`

Yeah, attributing file to a single person nowadays is bending the reality quite a lot